dmutils/email.py

```diff
@@ -1,17 +1,18 @@
 import base64
 import hashlib
-import six
-import struct
 import json
+import struct
+from collections import OrderedDict
 from datetime import datetime
 
+import itsdangerous
```

Comment:
I've also come across codebases that separate out the web framework specific imports into their own thing but this is probably overkill here

```
import base64
import hashlib
import json
import struct
from collections import OrderedDict
from datetime import datetime

import itsdangerous
import six
from cryptography import fernet

from flask import current_app
from flask._compat import string_types
from mandrill import Mandrill, Error

from notifications_python_client import NotificationsAPIClient
from notifications_python_client.errors import HTTPError

from .formats import DATETIME_FORMAT
```